ocs_ci/utility/utils.py

```diff
@@ -381,7 +386,7 @@ def run_cmd(cmd, **kwargs):
         log.warning(f"Command warning:: {r.stderr.decode()}")
     if r.returncode:
         raise CommandFailed(
-            f"Error during execution of command: {cmd}."
+            f"Error during execution of command: {'quiet command' if quiet_state else cmd}."
```

Comment:
Hmm, if we raise the exception anyway and there will be:
Error during execution of command: quiet command  , why not write also whole command if we do have such message in exception. I don't see any benefit to not to have mentioned cmd.

If I would like to avoid of raising the exception then I would add another argument ignore_error and than raise this exception only when not ignore_error.

Reply:
So here's the thing - even if the command failed, I wouldn't want the credentials to be exposed. However, completely muting all error logs for quiet commands will be confusing and will not reflect what truly happens behind the scenes.
For example:
```Writing testfile to bucket...
Error during execution of command: quiet command```
VS.
```Writing testfile to bucket...```
We have no way of knowing whether the command even succeeded, and if it fails - we have a rough idea of what the command **should've** been.

Reply:
Ah I see now what is the purpose here now.

I would choose different approach here, WDYT about some another parameter which will be called masked_secrets (list), you will need to define some another variable like masked_cmd inside this function and do something like this:

```python
masked_cmd = cmd
for secret in masked_secrets:
    masked_cmd.replace(secret, '*****')

then everywhere you will log or raise exception you will use this masked_cmd.  This case you will not leak secret data.   You can also make sure that you will mask you secret while returning value on line 393.   So the best way will be to cover the mentioned code in function ;) and just reuse it here and allow to use to use this function on different places where they need to mask their secret data.

Reply:
Great idea, done :)